vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/SqlResultBase.java

```diff
@@ -43,6 +46,24 @@ public int size() {
     return size;
   }
 
+  @Override
+  public <V> V property(PropertyKind<V> property) {
+    if (property == null) {
+      throw new IllegalArgumentException("Property can not be null");
+    }
+    if (properties == null) {
+      return null;
+    } else {
+      Object value = properties.get(property);
+      Class<V> type = property.type();
+      if (type == null) {
```

Comment:
no need for this,I think it's better to have an NPE

Reply:
an NPE on type.cast()

Reply:
see https://github.com/eclipse-vertx/vertx-sql-client/pull/392/commits/dbb4d080eff0dae1f078efa384d72a1a2aadd162.